internal/cephfs/volumemounter.go

```diff
@@ -229,12 +229,13 @@ func mountFuse(ctx context.Context, mountPoint string, cr *util.Credentials, vol
 		"-c", util.CephConfigPath,
 		"-n", cephEntityClientPrefix + cr.ID, "--keyfile=" + cr.KeyFile,
 		"-r", volOptions.RootPath,
-		"-o", "nonempty",
 	}
 
+	mntOpt := "nonempty"
```

Comment:
Is `nonempty` a valid mount option? This looks like a very ugly hack.

Reply:
Not added now, it was added in https://github.com/ceph/ceph-csi/pull/55

Reply:
hmm, ok